nb-apis/nb-api/pom.xml

```diff
@@ -161,6 +161,12 @@
             <artifactId>sqlite-jdbc</artifactId>
             <version>3.46.0.0</version>
         </dependency>
+      <dependency>
+        <groupId>io.qdrant</groupId>
+        <artifactId>client</artifactId>
+        <version>1.9.1</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
This does not belong in an API module. Any vendor or driver-specific dependencies need to be in their respective modules.

Reply:
This came from one of your changes